presto-product-tests/src/main/java/io/prestosql/tests/hive/TestTextFileHiveTable.java

```diff
@@ -36,59 +38,66 @@
     @Inject
     private HdfsClient hdfsClient;
 
+    @Inject
+    @Named("databases.hive.warehouse_directory_path")
+    private String warehouseDirectory;
+
     @BeforeTestWithContext
     public void setup()
             throws Exception
     {
-        hdfsClient.createDirectory("/user/hive/warehouse/TestTextFileHiveTable/single_column");
+        hdfsClient.createDirectory(warehouseDirectory + "/TestTextFileHiveTable/single_column");
```

Comment:
We can use `format` here to interpolate `warehouseDirectory`.

Reply:
I think this is fine. Single concatenation is more readable than format imo.